repository-data/application/src/main/resources/hcm-config/namespaces/hee/report.yaml

```diff
@@ -56,37 +48,65 @@ definitions:
             hipposysedit:path: hee:pageLastNextReview
             hipposysedit:primary: false
             hipposysedit:type: hee:pageLastNextReview
+            hipposysedit:validators: [required]
           /rightHandBlocks:
             jcr:primaryType: hipposysedit:field
             hipposysedit:multiple: true
             hipposysedit:ordered: true
             hipposysedit:path: hee:rightHandBlocks
             hipposysedit:type: hippo:compound
             hipposysedit:validators: [contentblocks-validator]
-          /logoGroup:
+          /string:
             jcr:primaryType: hipposysedit:field
             hipposysedit:mandatory: false
             hipposysedit:multiple: false
             hipposysedit:ordered: false
-            hipposysedit:path: hee:logoGroup
+            hipposysedit:path: hee:subtitle
             hipposysedit:primary: false
-            hipposysedit:type: hippo:mirror
+            hipposysedit:type: String
             hipposysedit:validators: [optional]
-          /microHero:
```

Comment:
I understand that the spec doesn't contain `microHero` field ? Could we confirm with Jon about its removal just to be sure [as we have this field in most of the other page DocumentTypes] ?